newrelic/build.go

```diff
@@ -85,6 +85,14 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		}
 	}
 
+	if _, ok, err := pr.Resolve("new-relic-python"); err != nil {
```

Comment:
```suggestion
	if _, ok, err := pr.Resolve("new-relic-python-config"); err != nil {
```

Needs to match detect.